apps/website/package.json

```diff
@@ -10,7 +10,7 @@
 		"build:prod": "yarn workspaces foreach -ptR run build && yarn build:copy_readme && yarn build:css && yarn build:next",
 		"build:next": "next build",
 		"build:css": "yarn generate:css",
-		"build:search_indicies": "yarn node scripts/generateAllIndicies.js",
```

Comment:
If you fix the name of the file to call here you also need to rename the actual file